.github/workflows/ci.yml

```diff
@@ -5,6 +5,11 @@ on:
   workflow_dispatch:
 
   pull_request:
+    paths-ignore:
+      - 'docs/**'
```

Comment:
So we do have a readthedocs build that reports a status back to GitHub, but that's not triggered by the ci.yml file here. What would happen in the hypothetical situation where someone makes a change in `docs` that breaks our documentation builds?

Reply:
The readthedocs build will still trigger on every push to an open PR and if breaking changes are made to the docs then that will be reported as a large red :x: by the readthedocs check.

I can make an example on another repo to show the behaviour

Reply:
No that's ok -- I'll take your word :smile: